CHANGELOG.md

```diff
@@ -5,10 +5,12 @@
 - Self-hosted etcd: if `etcd-hosts.checkpoint` file exists under `${datadir}/`,
   etcd pod will restore the hosts mapping from it before etcd bootstraps.
 - Add static TLS support for self-hosted etcd mode.
+- Added an RBAC rule for the resource `events` to allow the operator to create events.
```

Comment:
Just verified that the operator does not fail in normal cluster operations of creating, resizing etc if the rbac rule for events is not present.
It only fails to create the event.